### PR TITLE
Unused imports

### PR DESCRIPTION
**Pull request by tobolar**
Delete unused imports and use existing imports

### CODE REVIEW

File: Modelica_LinearSystems2/Controller/Internal/DiscreteStateSpace.mo [hunk 1]
```diff
@@ -129,7 +129,7 @@ initial equation
   elseif init == Types.Init.InitialOutput and nx>0 then
 //  y=y_start;
 //  xd[ny+1:nx]=[zeros(nx-ny,ny),identity(nx-ny)]*Modelica.Math.Matrices.solve(identity(nx)-discreteSystem.A,discreteSystem.B*u);
-      xd = Modelica.Math.Matrices.equalityLeastSquares(identity(nx)-discreteSystem.A, -discreteSystem.B*u, discreteSystem.C, y_start - discreteSystem.D*u);
+      xd = Matrices.equalityLeastSquares(identity(nx)-discreteSystem.A, -discreteSystem.B*u, discreteSystem.C, y_start - discreteSystem.D*u);
```
Comment (dietmarw) [inline Modelica_LinearSystems2/Controller/Internal/DiscreteStateSpace.mo:133]: Are you sure you want to remove absolute qualifiers like this when you are accessing classes from outside `Modelica_LinearSystems2`. There lead to dangerous wrong lookups.

File: Modelica_LinearSystems2/Internal/frequencyEvaluate.mo [hunk 2]
```diff
@@ -4,7 +4,6 @@ encapsulated function frequencyEvaluate
   import Modelica;
```
[the remark this answers is not in the document]
Comment (dietmarw) [reply Modelica_LinearSystems2/Internal/frequencyEvaluate.mo:4]: never mind. Did not see it is encapsulated.

[the remark this answers is not in the document]
Comment (tobolar) [reply Modelica_LinearSystems2/Internal/frequencyEvaluate.mo:4]: No Problem. Inspired by your comment I still have found some more meaningless imports of Modelica;

File: Modelica_LinearSystems2/Controller/Internal/DiscreteStateSpace.mo (hunk 1 above)
Comment (tobolar) [reply Modelica_LinearSystems2/Controller/Internal/DiscreteStateSpace.mo:133]: There is "import Modelica.Math.Matrices;" in that class. Would this be a case even if such an import exists?

Comment (dietmarw) [reply Modelica_LinearSystems2/Controller/Internal/DiscreteStateSpace.mo:133]: Ah ok in that case it's probably fine.

Comment (tobolar) [reply Modelica_LinearSystems2/Controller/Internal/DiscreteStateSpace.mo:133]: Phew, I'm relieved! Anyway, thanks for your review!

**Pull request merged**